cli/main.py

```diff
@@ -69,8 +84,9 @@ def __init__(self):
         sim_mode = 'simulation' if self.simulation_mode else 'live'
         click.echo("Running in {} {} mode".format(sim_mode, self.operating_mode))
 
-        self.payload = parse_nucypher_ini_config(filepath=self.config_filepath)
-        click.echo("Successfully parsed configuration file")
+        if use_config is True:
```

Comment:
Can this not be moved under the condition on L70?

Reply:
This is deprecated upstream, it was just a holdover for the hackathon.

Reply:
:+1: